x-pack/plugins/files/public/components/image/image.tsx

```diff
@@ -32,6 +32,15 @@ export interface Props extends ImgHTMLAttributes<HTMLImageElement> {
    * Emits when the image first becomes visible
    */
   onFirstVisible?: () => void;
+
+  /**
+   * As an optimisation images are only loaded when they are visible.
+   * This setting overrides this behavior and loads an image as soon as the
+   * component mounts.
+   *
+   * @default false
+   */
+  loadImageEagerly?: boolean;
```

Comment:
nit: maybe we could call this prop something simple, like `lazy`, which is `true` by default

```suggestion
  lazy?: boolean;
```

Reply:
yeah, to be honest, I was hoping we would be able to remove it soon so I wanted to pick an "ugly" name. but happy to use your suggestion for now